utils/abilities.cfg

```diff
@@ -300,7 +300,9 @@ The amount of damage dealt:"+" "+{DMG}
     [damage]
         id=eoma_areaeffectrandom
         name= _ "random area effect"
-        description=_"When this attack is used, random units adjacent to the target (except the attacker) get half of the basic damage. Does not work on defense."
+        description=_"When this attack is used, random ({CHANCETOHIT}%) units adjacent to the target (except the attacker) get half of the basic damage.
```

Comment:
I think this (and other similar descriptions in this commit) may trigger "MACRO inside translatable string" warning, see: #33 The solution would be to rewrite such descriptions so that `{CHANCETOHIT}` appears after translatable string and is added to the description using `+`. 
Example: `"When this attack is used, random units adjacent to the target (except the attacker) get half of the basic damage.
The chance to hit adjacent units is constant and equals:"+" "+{CHANCETOHIT}+"%"`